.github/workflows/odh-konflux-release-onboarder.yml

```
@@ -97,31 +97,63 @@ jobs:
+          else
+            if [ -z "$VERSION" ]; then
+              echo "ERROR: Version must be provided for Release builds"
+              exit 1
+            fi
+            echo "EFFECTIVE_TAG=$VERSION" >> $GITHUB_ENV
+            echo "APPLICATION_NAME=opendatahub-release" >> $GITHUB_ENV
+          fi
+
+
       - name: Prepare Tekton files
         run: |
```

actionlint (1.7.11)

[error] property "build_type" is not defined in object type {build_branch: string; component: string; pr_target_branch: string; version: string}

(expression)